src/Components/Results/Tabs/Tabs.tsx

```diff
@@ -7,11 +10,27 @@ type ResultsTabsProps = {
 const ResultsTabs = ({ currentTab }: ResultsTabsProps) => {
   const { uuid } = useParams();
 
+  const { programs, needs } = useResultsContext();
+
   return (
-    <>
-      <Link to={`/${uuid}/results/benefits`}>Benfits</Link>
-      <Link to={`/${uuid}/results/near-term-needs`}>Needs</Link>;
-    </>
+    <Grid container className="results-tab-container">
+      <Grid item xs={6} className="results-tab">
+        <NavLink to={`/${uuid}/results/benefits`} className={({ isActive }) => (isActive ? 'active' : '')}>
+          <FormattedMessage
+            id="resultsOptions.longtermBenefits"
+            defaultMessage={`Long-Term Benefits (${programs.length})`}
+          />
```

Comment:
Because `programs.length` is dynamic, we need to have it out of the `FormattedMessage`.
```suggestion
          <FormattedMessage
            id="resultsOptions.longtermBenefits"
            defaultMessage={'Long-Term Benefits '}
          />
          ({programs.length})
```